meta-mbl-bsp-common/recipes-bsp/atf/atf.inc

```diff
@@ -57,14 +57,15 @@ ATF_COMPILE_FLAGS = "-C ${S} \
       CRTTOOL=${S}/tools/cert_create/cert_create \
       CRTTOOLPATH=${S}/tools/fiptool \
       GENERATE_COT=1 \
-      LOG_LEVEL=40 \
       MBEDTLS_DIR=${WORKDIR}/mbedtls \
       PLAT=${PLATFORM} \
       TRUSTED_BOARD_BOOT=1 \
       USE_TBBR_DEFS=1 \
       FIP_ROM_OFFSET=${MBL_FIP_ROM_OFFSET} \
 "
 
+ATF_COMPILE_FLAGS += "${@bb.utils.contains("PACKAGECONFIG", "silent", " LOG_LEVEL=0", " LOG_LEVEL=40", d)}"
```

Comment:
Please, before this line adds the following otherwise bitbake will print a warning about invalid PACKAGECONFIG setting:
`PACKAGECONFIG[silent] = ""`

Reply:
Fixed

Reply:
`PACKAGECONFIG[silent] = ""` is still missing.